src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/FallThroughCheck.java

```diff
@@ -172,15 +190,36 @@
  * Example:
  * </p>
  * <pre>
- * switch (i) {
- *   case 1:
- *     i++;
- *     // FALL-THROUGH
```

Comment:
Please update config to be `name="reliefPattern" value="no break by design"`

And please add such comment somewhere in Example1 , to let it be violation by default, but when we use such value for property, it becomes no a violation.